include/ppi_omp/pipeline_omp.h

```diff
@@ -266,6 +264,7 @@ void pipeline(parallel_execution_omp p, FuncIn const & in, Stages ... sts ) {
                 }
             }
             //Create next stage
+            //stages(p, q, std::forward<Stages>(sts) ... );
```

Comment:
If I use the forward at this point, there are compilation problems for the examples pipeline+filter_test_ppi_example_OMP